tests/libfixmath/Makefile

```
@@ -2,6 +2,7 @@ include ../Makefile.tests_common
 
 USEPKG += libfixmath
 USEMODULE += libfixmath
+USEMODULE += xtimer
 
 TEST_ON_CI_WHITELIST += native
```

If it is working it can now be enabled on CI also.

You can inline in the squash

```suggestion
TEST_ON_CI_WHITELIST += all
```